press/press/doctype/site/site.json

```diff
@@ -596,6 +597,12 @@
    "fieldname": "additional_system_user_created",
    "fieldtype": "Check",
    "label": "Additional System User Created"
+  },
+  {
+   "description": "This key will be used to validate requests from saas site for billing & configuration purpose",
+   "fieldname": "saas_fc_communication_secret",
```

Comment:
No need to use FC in name. It could simply be `saas_communication_secret` or `saas_secret` to make it more simple.